docs/validations/validations-rules-reference.mdx

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_label: 'Rules Reference'
+---
+
+# Rules Reference
+
+Rules can be asserted against a gameplay recording to ensure that the recording meets specific criteria. We provide a set of rule types that can be used to construct assertions based on various performance metrics.
+
+Rules belong to a Validation Suite, and are asserted each time the Validation Suite is run.
+
+## Rule Types
+
+### Frames Per Second (with percentile)
+
+Assert that the frames per second of the gameplay recording are above a threshold, within a specified percentile.
```

Comment:
These descriptions need to account for the fact that meeting the threshold does not violate it. 

"are above a threshold" -> "do not fall below the threshold"
similar for the other rule descriptions

Reply:
I agree. I have the 3 descriptions containing either "above a threshold", or "below a threshold"

Reply:
```suggestion
Assert that the frames per second of the gameplay recording do not fall below a threshold, within a specified percentile.
```

Reply:
ah there was some confusion on my part regarding your original comment. Changes made

Reply:
f